minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/ShutdownHook.java

```diff
@@ -55,10 +55,12 @@ public void run() {
 
         System.out.println("Initiating shutdown of bootstrap change ingestors...");
         ConfigurationChangeCoordinator notifier = runner.getChangeCoordinator();
-        try {
-            notifier.close();
-        } catch (IOException ioe) {
-            System.out.println("Could not successfully stop notifier due to " + ioe);
+        if (notifier != null) {
```

Comment:
After attempting to start without setting the file path I see the following message printed to the terminal ~20 seconds after the initial "start" command ends (and gives control back to the user). Is there a way to check if the MiNiFi instance was ever started in the first place before attempting to stop it?
```
MiNiFi has not finished shutting down after 20 seconds. Killing process.
MiNiFi is done shutting down
```